src/js/manager.js

```diff
@@ -361,69 +400,126 @@ function handlePreview(indexArr) {
       return;
     }
     chrome.tabs.query({"currentWindow": true}, function (tabs) {
-      var saveListURL = { "URL": []};
+      var saveList = [];
       for (var i = 0; i < tabs.length; i++) {
         if (tabs[i].title.indexOf(keyword) > -1) {
-          saveListURL.URL.push(tabs[i].url);
+          saveList.push({url : tabs[i].url, title : tabs[i].title });
         }
       }
-      saveUrlToLocalStorage(saveListURL);
+      saveUrlToLocalStorage(saveList);
     });
-  } else if (option == CONST_VIEW){
-    /*임시로...*/
-    var viewlist = '';
-    for(var i = 0; i < localStorage.length; i++){
-      viewlist += localStorage.key(i) + '\n';
-    }
-    alert(viewlist);
-
-    if(keyword === 'delete'){
-      localStorage.clear();
-    }
   }
 }
 
-function saveUrlToLocalStorage(saveListURL){
-  if (saveListURL.URL.length != 0) {
+function saveUrlToLocalStorage(saveList){
+  if (saveList.length != 0) {
     var saveListName = prompt("Please enter name for save list", "NewList");
-    if(localStorage.getItem(saveListName) != null){
+    if (localStorage.getItem(saveListName) != null) {
       alert("Already Exist! Please enter another name.");
       return;
     }else{
-      localStorage.setItem(saveListName, JSON.stringify(saveListURL));
-      //alert(JSON.stringify(saveListURL));
+      localStorage.setItem(saveListName, JSON.stringify(saveList));
     }
-  } else {
+  }
+  else {
     $('#error-message').text('no matched tabs');
   }
 }
+
+/*
+ * function for merge
+ */
+function handleMerge(option, keyword) {
+  if (emptyKeyword(keyword)) {
+    return;
+  }
+  var url = 'src/html/merge.html?option=' + option + '&keyword=' + keyword;
+  chrome.tabs.create({"url": url, "selected": true});
+}
+
+/*
+* function for auto complete keyword input box
+*/
+function autocomplete(){
+  var command = $('#command').val();
+  var option = $('#option').val();
+  $('#keyword').autocomplete();
+  if(command == 'open' && option == CONST_SAVED){
+    var openSaveSource = [];
+    for(var i = 0; i < localStorage.length; i++){
+      openSaveSource.push(localStorage.key(i));
+    }
+
+    $( '#keyword' ).autocomplete({
+      source : openSaveSource,
+      minLength : 0,
+      position: { my : "right top", at: "right bottom", collision : "fit"},
+    }).on("focus", function(){
+      $(this).autocomplete("search", '');
+    });
+  } else if(command == 'search' && option == CONST_TITLE){
+    var openSaveSource = [];
+
+    chrome.tabs.query({"currentWindow": true}, function (tabs) {
+      for(var i=0; i<tabs.length; i++){
+        openSaveSource.push(tabs[i].title);
+      }
+    });
+
+    $( '#keyword' ).autocomplete({
+      source : openSaveSource,
+      minLength : 0,
+      position: { my : "right top", at: "right bottom", collision : "fit"},
+    }).on("focus", function(){
+      $(this).autocomplete("search", '');
+    });
+  } else {
+    $('#keyword').autocomplete("destroy");
+  }
+}
+
+function showKeywordBox(){
```

Comment:
Please leave comments in all functions if you have time. It's a little hard to understand without comments.